src/main/java/propertyFiles/Bundle.properties

```diff
@@ -1620,10 +1621,13 @@ file.ingestFailed.header=Upload Completed with Errors
 file.ingestFailed.message=Tabular data ingest failed.
 file.downloadBtn.format.all=All File Formats + Information
 file.downloadBtn.format.tab=Tab-Delimited
-file.downloadBtn.format.original=Original File Format ({0})
-file.downloadBtn.format.rdata=RData Format
+file.downloadBtn.format.original={0} (Original File Format)
+file.downloadBtn.format.rdata=RData
 file.downloadBtn.format.var=Variable Metadata
 file.downloadBtn.format.citation=Data File Citation
+file.downloadBtn.restricted=Restricted
+file.downloadBtn.restricted.with.access.granted=Restricted with Access Granted
```

Comment:
We can now get rid of these and just use:
restricted=Restricted
restrictedaccess=Restricted with Access Granted